Claudia/src/main/java/com/endava/task/second/Item.java

```diff
@@ -0,0 +1,23 @@
+package com.endava.task.second;
+
+public class Item {
+    private static long countId = 1;
```

Comment:
Could you please also try java.util.Random in order to randomize id (also change this logic in Order class and User class)

Reply:
Done.